### DISCUSSION TITLE
Changing Page & Changing number of rows to be shown return nothing

### DISCUSSION THREAD

**Discussion by BenOussama180** [Power-Components/livewire-powergrid#713] [BUG]
when i change page and change the per page values in the dropdown menu i lose all the data that was shown
![image](https://user-images.githubusercontent.com/81111919/200323763-bc711146-fcfb-4c1f-bcc5-16c841ccf8e7.png)

**Comment by luanfreitasdev** [comment]
Did this happen before or after an update?

**Reply by BenOussama180** [reply]
i updated powergrid to 3.x like 1 month ago but i just noticed this bug
